### PR TITLE
4.4.5 5.1.1

### PR DESCRIPTION
**Pull request by MacJoom**
4.4.5 5.1.1

### CODE REVIEW

File: www/core/j4/default.xml [hunk 1]
```diff
@@ -55,21 +55,21 @@
...
 		<type>file</type>
-		<version>4.4.4</version>
-		<infourl title="Joomla 4.4.4 Release">https://www.joomla.org/announcements/release-news/5907-joomla-5-1-0-and-joomla-4-4-4-are-here.html</infourl>
+		<version>4.4.5</version>
+		<infourl title="Joomla 4.4.5 Release">https://www.joomla.org/announcements/release-news/5908-joomla-5-1-1-and-joomla-4-4-5-are-here.html</infourl>
 		<downloads>
-			<downloadurl type="full" format="zip">https://downloads.joomla.org/cms/joomla4/4-4-4/Joomla_4.4.4-Stable-Update_Package.zip</downloadurl>
-			<downloadsource type="full" format="zip">https://github.com/joomla/joomla-cms/releases/download/4.4.4/Joomla_4.4.4-Stable-Update_Package.zip</downloadsource>
-			<downloadsource type="full" format="zip">https://update.joomla.org/releases/4.4.4/Joomla_4.4.4-Stable-Update_Package.zip</downloadsource>
+			<downloadurl type="full" format="zip">https://downloads.joomla.org/cms/joomla4/4.4.5/Joomla_4.4.5-Stable-Update_Package.zip</downloadurl>
```
Comment (zero-24) [inline www/core/j4/default.xml:61]:
```suggestion
			<downloadurl type="full" format="zip">https://downloads.joomla.org/cms/joomla4/4-4-5/Joomla_4.4.5-Stable-Update_Package.zip</downloadurl>
```

File: www/core/j4/default.xml [hunk 2]
```diff
@@ -80,21 +80,21 @@
...
```
Comment (zero-24) [inline www/core/j4/default.xml:86]:
```suggestion
			<downloadurl type="full" format="zip">https://downloads.joomla.org/cms/joomla4/4-4-5/Joomla_4.4.5-Stable-Update_Package.zip</downloadurl>
```

File: www/core/sts/extension_sts.xml [hunk 3]
```diff
@@ -24,12 +24,12 @@
...
 		<type>file</type>
-		<version>4.4.4</version>
-		<infourl title="Joomla 4.4.4 Release">https://www.joomla.org/announcements/release-news/5907-joomla-5-1-0-and-joomla-4-4-4-are-here.html</infourl>
+		<version>4.4.5</version>
+		<infourl title="Joomla 4.4.5 Release">https://www.joomla.org/announcements/release-news/5908-joomla-5-1-1-and-joomla-4-4-5-are-here.html</infourl>
 		<downloads>
-			<downloadurl type="full" format="zip">https://downloads.joomla.org/cms/joomla4/4-4-4/Joomla_4.4.4-Stable-Update_Package.zip</downloadurl>
-			<downloadsource type="full" format="zip">https://github.com/joomla/joomla-cms/releases/download/4.4.4/Joomla_4.4.4-Stable-Update_Package.zip</downloadsource>
-			<downloadsource type="full" format="zip">https://update.joomla.org/releases/4.4.4/Joomla_4.4.4-Stable-Update_Package.zip</downloadsource>
+			<downloadurl type="full" format="zip">https://downloads.joomla.org/cms/joomla4/4.4.5/Joomla_4.4.5-Stable-Update_Package.zip</downloadurl>
```
Comment (zero-24) [inline www/core/sts/extension_sts.xml:30]:
```suggestion
			<downloadurl type="full" format="zip">https://downloads.joomla.org/cms/joomla4/4-4-5/Joomla_4.4.5-Stable-Update_Package.zip</downloadurl>
```

**Comment by zero-24** [comment]
The URLs are still not fully correct, please comment check my comments

**Comment by MacJoom** [comment]
Thanks - I hope they are correct now

**Comment by zero-24** [comment]
Please also check the `<targetplatform name="joomla" version="5.0"/>` for the 5.1.1 downloads. That should be `<targetplatform name="joomla" version="5.[01]"/>` cant suggest that as its out of the scope of the diff right now.

**Pull request merged**